src/test/java/io/redisearch/CreateTest.java

```diff
@@ -34,23 +34,13 @@ public void allOptions() throws Exception {
     defaultOptions.serializeRedisArgs(arrayList);
 
     assertEquals(
-        Arrays.asList("ON", "HASH", "NOOFFSETS", "NOFIELDS", "NOFREQS", "TEMPORARY", "1234", "STOPWORDS", "2", "stop", "run"),
+        Arrays.asList("NOOFFSETS", "NOFIELDS", "NOFREQS", "TEMPORARY", "1234", "STOPWORDS", "2", "stop", "run"),
         arrayList);
   }
 
-  @Test
```

Comment:
@sazzad16 why did you delete the default empty test?

Reply:
@gkorland This test checking if `ON HASH` is being added in FT.CREATE command arguments. But I'm changing this behavior in this PR.

RediSearch accepts and processes FT.CREATE command without ON option. So empty constructor should not send any ON option to RediSearch, rather should depend on RediSearch's default behavior. This is what I'm doing in this PR and as a result this test won't work.

Note-1: Currently, RediSearch works exactly same as without ON option and with ON HASH.

Note-2: This change does not change any client behavior.